docs/reference/commandline/update.md

```diff
@@ -31,44 +31,25 @@ Options:
 ```
 
 The `docker update` command dynamically updates container configuration.
-You can use this command to prevent containers from consuming too many resources
-from their Docker host.  With a single command, you can place limits on
-a single container or on many. To specify more than one container, provide
-space-separated list of container names or IDs.
+you can Use this command to prevent containers from consuming too many 
```

Comment:
Oh, that's weird

Oh! Is see what happened, I "synced" with the man page, and that contained this